spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/ConfigMapsMixedTests.java

```diff
@@ -54,17 +55,13 @@
 
 	protected static final String APPLICATION_NAME = "configmap-mixed-example";
 
-	@ClassRule
```

Comment:
this is now injected by `@EnableKubernetesMockClient(crud = true, https = false)` so no need for the server. the rest of the changes are trivial, imho, here.